tox.ini

```
@@ -1,11 +1,10 @@
 [tox]
-envlist = py{38,311,312},pipdeptree{,-requirements},flake8,report
+envlist = py{311,312}-{pipdeptree,pipdeptree-requirements,flake8},report
```

This way you disable the `manage.py test` tests; that's why the `coverage` reports are empty.

This should read:

```suggestion
envlist = py3{11,12}{,-pipdeptree,-pipdeptree-requirements,-flake8},report
```
```

---

I see, thanks!